src/xdocs/checks/naming/packagename.xml.template

```diff
@@ -34,33 +34,35 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p id="Example1-config">
+        <p id="resources\com\puppycrawl\tools\checkstyle\checks\naming\packagename\Example1-config">
           To configure the check:
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example1.txt"/>
+                 value="resources\com\puppycrawl\tools\checkstyle\checks\naming\packagename\Example1.java"/>
```

Comment:
Do not change `/` to `\`.
Just change extension from txt to java

Reply:
I'm sorry, I solved this but still can't pass CI/CD.